root.spec

```diff
@@ -56,6 +57,7 @@ cmake ../%{n}-%{realversion} \
   -Dhdfs=OFF \
   -Dqt=OFF \
   -Dtmva=ON \
+  -DPython3_EXECUTABLE="${PYTHON3_ROOT}/bin/python3" \
```

Comment:
why we are building with python3 here while CPYY_BACKEND_LIBRARY points to libcppyy_backend2_7.so?

Reply:
it uses both installations. the build fails if I don't provide the python3 executable location. reading the new cmake configs it seems the idea is to 'find out which python are available' 2, 3 or both and, use them nevertheless when I used only python2 it complained right away for incomplete cmake config in here:
https://github.com/root-project/root/blob/master/cmake/modules/SearchInstalledSoftware.cmake#L1490
In general I wanted to have a working build and then ask questions 'can we only use python2 if we want to'

Reply:
ah ok, I thought it was only building for python3. If it is build for both then this looks good. 
Can we build yoda also for both python version?

Reply:
I changed the CPYY_BACKEND_LIBRARY to libcppyy_backend3_8 
and few other things to try. We can see also if pyroot builds with py3 (which is available for py3 and should work)